api/src/controllers/extensions.ts

```diff
@@ -45,10 +45,7 @@ router.get(
 		}
 
 		res.setHeader('Content-Type', 'application/javascript; charset=UTF-8');
-		res.setHeader(
-			'Cache-Control',
-			env.EXTENSIONS_CACHE_TTL ? getCacheControlHeader(req, ms(env.EXTENSIONS_CACHE_TTL as string)) : 'no-store'
-		);
+		res.setHeader('Cache-Control', getCacheControlHeader(req, ms(env.EXTENSIONS_CACHE_TTL as string), false, false));
```

Comment:
@paescuj when I was testing out the production build for #17530 (because dev doesn't load from the `/extensions/sources/index.js` API route), I just noticed that this removal of ternary operator will make it pass an undefined `EXTENSIONS_CACHE_TTL` to `ms()`, which causes an error:

![](https://user-images.githubusercontent.com/42867097/219327212-31aaf1a6-bbf5-4f81-a98d-4f4d655b602a.png)

whereas `ASSETS_CACHE_TTL` doesn't have this issue since there's a default value:

https://github.com/directus/directus/blob/89fbac36ceaea6066ecfeaef25f6923ea4180bdb/api/src/env.ts#L252

however I've also noticed your PR #17498 which should technically solve this, so just wanted to bring this up to make sure #17498 gets merged before the next release 👍

Reply:
@azrikahar That's right, thanks for bringing this up! This pull request depends on https://github.com/directus/directus/pull/17498 which should definitely be merged before the next release.